packages/core/helper-plugin/src/components/tests/PageSizeURLQuery.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('PageSizeURLQuery', () => {
 
   describe('interaction', () => {
     it('should change the value when the user selects a new value', async () => {
-      let testLocation = null;
+      let testLocation: { search?: string } = {};
```

Comment:
```suggestion
      let testLocation = null!;
```

Will also mean you can stop checking if it's a thing etc. because you _know_ it'll be defined via the test 👍🏼